CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Code of Conduct
```

Comment:
Small thought on file naming: Would `CODE_OF_CONDUCT.md` be more clear?

Reply:
🤷‍♀️ We are going to put it up on the website at /code-of-conduct and link prominently to that, so I think it doesn't matter too much in the repo? 

I've seen it many ways, including as it is now

Reply:
👍